docs/processors/standalone/wasm.mdx

```diff
@@ -0,0 +1,259 @@
+---
```

Comment:
@lovromazgon I just realized it'd be probably better if we renamed this page to `how-it-works.mdx`